.github/workflows/build-test.yml

```diff
@@ -210,3 +214,33 @@ jobs:
                 CLOUD_PROVIDER: ${{ matrix.cloud }}
                 TARGET_DOCKER_TEST_IMAGE: ${{ matrix.image }}
               run: ./ci/test.sh
+
+    display_code_coverage:
+        needs: test-mac-required
+        name: Display code coverage
+        runs-on: macos-latest
+        strategy:
+          matrix:
+            cloud: [ 'AWS' ]
```

Comment:
we don't need it because we are not connecting to Snowflake

Reply:
removed